app/models/ingredient.rb

```diff
@@ -0,0 +1,6 @@
+class Ingredient < ApplicationRecord
+  validates :name, presence: true,
+    uniqueness: true,
+    format: { with: /\A[a-z][a-z\s]+[a-z]\z/,
+    message: "only allows underscore letters and whitespace" }
```

Comment:
there is no underscore in your regex. I think that the last part (`[a-z]`) is not needed.